README.md

```diff
@@ -108,6 +112,7 @@ Run a container:
 podman run -it \
            -e PRIVATE_KEY="$(cat <private key filename>)"
            --env-file=env.txt \
+           -v ~/runner.toml:/home/gitlab-runner/.gitlab-runner/config.toml
```

Comment:
I guess this line is not necessary for this PR.

Reply:
since I do not set UNREGISTER_ON_EXIT and REGISTER_ON_ENTER by default it makes perfectly sense to volume/persist the runner config by default.
This however depends on how you want to run it.